src/main/res/values/strings.xml

```diff
@@ -17,25 +17,25 @@
   <string name="anti_feature_non_free_dependencies">Depends on non-free applications</string>
   <string name="anti_feature_non_free_network">Promotes non-free network services</string>
   <string name="anti_feature_tracking">Tracks or reports your activity</string>
-  <string name="anti_feature_upstream_non_free">Upstream source code is not free</string>
+  <string name="anti_feature_upstream_non_free">Upstream source code is not libre</string>
   <string name="anti_features">Anti-features</string>
   <string name="app_name">Foxy Droid</string>
   <string name="application">Application</string>
   <string name="application_not_found">Application not found</string>
-  <string name="authors_website">Author\'s website</string>
-  <string name="authors_email">Author\'s email</string>
+  <string name="authors_website">Author website</string>
+  <string name="authors_email">Author e-mail</string>
```

Comment:
"email" is recommended to use instead of "e-mail"

Reply:
Not if you want the translations to be correct. Both are valid in English, but not elsewhere.

Reply:
But it's an English translation, right? I prefer "email" in English, and this is how it's recommended to name it in English.

Reply:
"email" is newer, and doesn't enjoy the same recognition. "email" is more prevalent, and it will end up being "mail" sooner or later. Other languages will never change.

Reply:
So, it's up to translations to translate "email" to their language. Even to "e-mail" if this is how it's translated to their language (for instance, in Russian it's "Электронная почта", which is so far from "e-mail", and I'm fine with it).

Reply:
People don't add it in error, they forget to include it if it is missing. https://da.wikipedia.org/wiki/E-mail https://pt.wikipedia.org/wiki/E-mail https://ia.wikipedia.org/wiki/Currero_electronic https://sv.wikipedia.org/wiki/E-post https://nn.wikipedia.org/wiki/E-post https://no.wikipedia.org/wiki/E-post https://fr.wikipedia.org/wiki/Courrier_électronique https://de.wikipedia.org/wiki/E-Mail and so on don't allow ~"email" and similar.

Reply:
But https://ru.wikipedia.org/wiki/Электронная_почта and https://en.wikipedia.org/wiki/Email (ofc). We are talking about __English__ translation now, and not how it will be translated to other language.

Reply:
It will be translated from English, where the error in translation is introduced, by forgetting the dash. Vice-versa, it doesn't get introduced, as the dash isn't carried over into words where it doesn't belong. If you are particular about esteem and professionalism, "e-mail" is the superior choice. Reverting.